notification/test/integration/notification.handler.spec.js

```diff
@@ -12,7 +12,7 @@ const notifications = require('../resources/notification')
 const localhostIp = address()
 
 describe('notification module', () => {
-  const ctpClient = ctpClientBuilder.get(config)
+  const ctpClient = ctpClientBuilder.get(config, ['manage_extensions:'.concat(config.ctp.projectKey)])
```

Comment:
Notification module does not need to manage extensions. This is required for extension module only.

Reply:
It does because of line 26 on integration-test-set-up.js which does a delete on all payments, types and extensions to ensure there is no interference with the integration tests.